deploy/pulse.conf

```diff
@@ -14,18 +5,18 @@ upstream production_pulse {
 }
 
 server {
-  listen 443 ssl spdy;
   server_name  pulse.cio.gov;
 
-  ssl_certificate      /etc/nginx/ssl/keys/pulse.cio.gov.chained.crt;
-  ssl_certificate_key  /etc/nginx/ssl/keys/pulse.cio.gov.key;
+  ssl_certificate      /etc/letsencrypt/live/pulse.cio.gov/fullchain.pem;
+  ssl_certificate_key  /etc/letsencrypt/live/pulse.cio.gov/privkey.pem;
```

Comment:
This change dates from when we moved pulse.cio.gov to use a certificate from Let's Encrypt.